apps/condo/domains/billing/schema/DiscoverServiceConsumersService.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Generated by `createservice billing.DiscoverServiceConsumersService --type mutations`
+ */
+
+const { get } = require('lodash')
+
+const { GQLCustomSchema, getById } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/billing/access/DiscoverServiceConsumersService')
+const { BillingAccount } = require('@condo/domains/billing/utils/serverSchema')
+const { Resident, ServiceConsumer } = require('@condo/domains/resident/utils/serverSchema')
+
+const DiscoverServiceConsumersService = new GQLCustomSchema('DiscoverServiceConsumersService', {
+    types: [
+        {
+            access: true,
+            type: 'input DiscoverConsumersServiceInput { dv: Int!, sender: SenderFieldInput!, address: String!, unitName: String!, unitType: String!, billingAccountId: ID, residentId: ID }',
+        },
+    ],
+    
+    mutations: [
+        {
+            access: access.canDiscoverConsumersService,
+            schema: 'discoverConsumersService(data: DiscoverConsumersServiceInput!): [ServiceConsumer]',
```

Comment:
Do we really need to return service consumers?
Our mutation will be used in hooks